operations/policies/policies_webhooks.rego

```diff
@@ -11,52 +11,141 @@ webhook_rule_present(obj, expected) if {
   rule.scope == "Namespaced"
 }
 
-validate_webhook(obj, name_prefix, path, side_effect, match_policy, rule) if {
-    object.get(obj.metadata.labels, "grafana.com/inject-rollout-operator-ca", "false") == "true"
-    object.get(obj.metadata.labels, "grafana.com/namespace", "") != null
-    startswith(obj.webhooks[0].name, name_prefix)
-    obj.webhooks[0].clientConfig.service.namespace == object.get(obj.metadata.labels, "grafana.com/namespace", "")
-    obj.webhooks[0].clientConfig.service.path == path
-    obj.webhooks[0].clientConfig.service.port == 443
-    obj.webhooks[0].failurePolicy in {"Fail","Ignore"}
-    object.get(obj.webhooks[0], "matchPolicy", "") == match_policy
-    obj.webhooks[0].sideEffects == side_effect
+# Assert that all ValidatingWebhookConfiguration have the expected inject-rollout-operator-ca label
+deny contains msg if {
+    obj := input[_].contents
+    obj.kind == "ValidatingWebhookConfiguration"
+    object.get(obj.metadata.labels, "grafana.com/inject-rollout-operator-ca", "false") != "true"
+    msg := sprintf("ValidatingWebhookConfiguration should have label grafana.com/inject-rollout-operator-ca=true, %v", [display_name(obj)])
+}
+
+# Assert that all ValidatingWebhookConfiguration clientConfig.service.namespace should match the grafana.com/namespace label
+deny contains msg if {
+    obj := input[_].contents
+    obj.kind == "ValidatingWebhookConfiguration"
+    obj.webhooks[0].clientConfig.service.namespace != object.get(obj.metadata.labels, "grafana.com/namespace", "")
+    msg := sprintf("ValidatingWebhookConfiguration clientConfig.service.name should match grafana.com/namespace label, %v. %v != %v", [display_name(obj), obj.webhooks[0].clientConfig.service.namespace, object.get(obj.metadata.labels, "grafana.com/namespace", "")])
+}
```

Comment:
This largely duplicates the similar rule for `MutatingWebhookConfiguration` - can we combine them somehow?

(similar feedback applies to other rules)